acceptance/openstack/autoscaling/v1/policies_test.go

```diff
@@ -0,0 +1,97 @@
+package v1
+
+import (
+	"testing"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/openstack/autoscaling"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/tools"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/autoscaling/v1/policies"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestPolicyLifecycle(t *testing.T) {
```

Comment:
=== RUN   TestPolicyLifecycle
    helpers.go:31: Attempting to create AutoScaling Group
    helpers.go:34: Created AutoScaling Group: a16eeb2c-590a-4ebc-a9d5-2cae0a98519a
    policies_test.go:29: Attempting to create AutoScaling Policy
    policies_test.go:46: Created AutoScaling Policy: 6690f92e-d256-40fe-828e-8f5afc856fa2
    policies_test.go:61: Attempting to update AutoScaling policy
    policies_test.go:90: Updated AutoScaling Policy
    tools.go:72: {
          "scaling_group_id": "a16eeb2c-590a-4ebc-a9d5-2cae0a98519a",
          "scaling_policy_name": "as-policy-update-YB0",
          "policy_status": "INSERVICE",
          "scaling_policy_type": "RECURRENCE",
          "alarm_id": "",
          "scheduled_policy": {
            "launch_time": "10:30",
            "recurrence_type": "Weekly",
            "recurrence_value": "1,3,5",
            "start_time": "",
            "end_time": "2040-12-31T10:30Z"
          },
          "scaling_policy_action": {
            "operation": "ADD",
            "instance_percentage": 30
          },
          "cool_down_time": 300,
          "create_time": "2022-11-04T14:39:38Z"
        }
    policies_test.go:49: Attempting to delete AutoScaling Policy
    policies_test.go:52: Deleted AutoScaling Policy: 6690f92e-d256-40fe-828e-8f5afc856fa2
    helpers.go:40: Attempting to delete AutoScaling Group
    helpers.go:45: Deleted AutoScaling Group: a16eeb2c-590a-4ebc-a9d5-2cae0a98519a
--- PASS: TestPolicyLifecycle (39.46s)
PASS


Debugger finished with the exit code 0